### DISCUSSION TITLE
Sign in Broken neovim after payment

### DISCUSSION THREAD

**Discussion by 0xmovses** [community/community#19107] [Copilot Conversations]
I signed up for the $10 a month plan. I run `:Copilot setup` and I enter the auth code successfully, when redirected back into Neovim, I get the error:

``Copilot: Authentication failure: You don't have access to Github Copilot. Join the waitlist by visiting https>://copilot.github.com``

**Comment by VapidLinus** [comment]
Same issue for me in IntelliJ

**Reply by 0xmovses** [reply]
everything was working fine for about two months. The Copilot said I needed to upgrade. I upgraded, opted in for the paid service, and now not working, so quite sure its nothing to do with my config.

**Comment by melvincwng** [comment]
@rvmelkonian, try contacting [Github Support](https://support.github.com/) to see if they can resolve your issue.

Hope this helps, thanks 😊

**Comment by ShinChven** [comment]
Really bad product quality control.

**Comment by gwd999** [comment]
similar issues here signed up for plan -> copilot no longer "wants to work" in neovim - constantly "Requesting Action" to fully configure or complete my setup. But when I call upon the "Action" to "Complete Settings" it just directs me to the Copilot Settings page of my Profile where tbh there aren't a lot of Settings (left) to complete? 
Any solutions to this problem ... 
In VSC all works without issues for me so I have the feeling some auth is not happening in neovim the way it was supposed to?     

OK IN MY CASE IT WAS MY FAULT I WAS SIGNED IN WITH A DIFFERENT github USER  on the neovim running in WSL2
So signing out and doing a `:Copilot auth` it gave me a fresh code to use to activate it correctly.

**Comment by VapidLinus** [comment]
I had to go to this page and complete my settings:

https://github.com/settings/copilot

**Reply by dave-budah** [reply]
Thank you, i almost trashed Github copilot.
